### PR TITLE
Added HTTP and HTTPS code examples

### PR DESCRIPTION
**Pull request by Akshaya101**
HTTP and HTTPS code examples have been added.

### CODE REVIEW

**Review by atapas — CHANGES_REQUESTED** [review]
@Akshaya101 

Here are some initial comments: https://github.com/atapas/nodeX/issues/1#issuecomment-1061387863

**Comment by Akshaya101** [comment]
HTTP module has been added. Let me know if any changes are to be made.

File: http-module/code/write-head.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//Syntax = writeHead(statusCode, statusMessage[Optional], headers)
+//Purpose = sends a response header to the request
+
+const http = require('http')
+http.createServer(function (req, res) {
+    // res.writeHead(200, {
```
Comment (atapas) [inline http-module/code/write-head.js:6]: @Akshaya101 remove the commented code

**Comment by atapas** [comment]
> HTTP module has been added. Let me know if any changes are to be made.

Thank you!

Can you please add an example for Create a https server as well as the PR says for both HTTP and HTTPS module? You can add a separate folder and readme for https like you did for http. Thanks a lot.

**Comment by Akshaya101** [comment]
HTTPS module has been added. Let me know if there are any changes to be made

**Comment by atapas** [comment]
> HTTPS module has been added. Let me know if there are any changes to be made.

Thanks a lot, @Akshaya101. Looks great. It will help a lot of developers.

**Review by atapas — APPROVED** [review]
Thanks @Akshaya101 RTM

**Pull request merged**